lte/gateway/c/session_manager/GrpcMagmaUtils.h

```diff
@@ -27,3 +27,5 @@ std::string get_env_var(std::string const& key);
 void PrintGrpcMessage(const google::protobuf::Message& message);
 
 std::string indentText(std::string basicString, int indent);
+
+void prepend_imsi_with_prefix(std::string& imsi);
```

Comment:
Do we want to modify the input string here or return a new one? up to you, but my comment above applies.

Reply:
I would opt for `std::string prepend_imsi_with_prefix(const std::string& imsi);` as output parameters can be confusing.

Reply:
Changed the function prototype to `std::string prepend_imsi_with_prefix(const std::string& imsi);` , so it takes input parameter and checks if the "IMSI" substring is present or not, if it isn't present, it returns `"IMSI" + imsi` else it simply returns `imsi`